### DISCUSSION TITLE
How to copy maps and favorites to new Android device ?

### DISCUSSION THREAD

**Discussion by rupertvdh** [osmandapp/OsmAnd#19002] [Q&A]
Dear specialists, I am using OsmAnd for some years now on 2 devices (smartphone and -old- tablet).
To update the maps, I use my PC to download the maps (at fiber-speed) from http://download.osmand.net/list.php.
I then copy them to both devices.

To replace my -old- tablet, I bought a new one :
Lenovo Tab M8 (4th Gen)
Model TB300FU
Android 13

After installing OmsAnd on it, I seem to have no access to the folder /storage/emulated/0/Android/net.osmand/files.
The pre-installed file-explorer app gives "nothing here" for "/Android/data".
Connected to my PC, I can go to "This PC\Lenovo Tab M8 (4th Gen)\Internal shared storage\Android\data\net.osmand\files" and see all files there, but I can not copy PC-files (maps or favorites) to it.
Via the PC, I can open and edit files on the new tablet (eg. the "favorites.gpx") but not save any modifications.

In other threads, I found that the "data storage folder" can be changed, so I moved it to
"/storage/emulated/0/Documents/net.osmand/files" and yes, I see all folders and files with the pre-installed file-explorer app.
BUT :
when I copy any map there, it is not shown in the OmsAnd application ...
When I copy my favorites.gpx to it, none of them is shown on the screen ...
All this even after a forced "stop application".

OK, so far so bad... 
When I create -via tha application-screen- some favorites on the first and only down-loaded map, the "favorites.gpx" in the Documents/net.osmand folder is not modified ???
And the original folder "Android\data\net.osmand\files\favorites" is empty. No favorites.gpx there.
Where-oh-where then are they stored ???

Right, next trial : use the "Export to file" option on the smartphone to make a copy of my favorites.
But this ends up in showing me the icons of all kind of apps.
When I chose "Notepad" hoping to create some kind of file I can import on the new tablet, nothing happens at all ...

How could I get the OsmAnd app up and running as I like it on this new tablet ???

**Comment by tommycrock** [comment]
To add the maps via PC, copy them to Downloads on the tablet, then Open with... Osmand from a file manager app on the tablet. This will get around the Android 13 restrictions. You might find it more convenient to download over WiFi.
For Favourites, use the Export to file option as you suggest and then choose an app that can handle the file like an email, file manager, instant messenger app. If you use a file manager again you'll have to save the file somewhere on the old tablet and manually transfer it. However you get it to the new tablet, click to Open with... Osmand or Import from file.
Hope that helps!

**Comment by rupertvdh** [comment]
It's different from what I'm used to, but it works fine.
All is now up and running.
Thanks for your help.

**Comment by rupertvdh** [comment]
Just to close the thread ...
